src/cli/commands/dht.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = {
+  command: 'dht <command>',
+
+  description: 'Issue commands directly through the DHT.',
+
+  builder (yargs) {
+    return yargs.commandDir('dht')
+  },
+
+  handler (argv) {
+  }
+}
```

Comment:
Is the DHT full implemented on http api and CLI as well? Mind updating https://github.com/ipfs/ipfs/blob/master/IMPLEMENTATION_STATUS.md#dht ?

Reply:
done in https://github.com/ipfs/ipfs/pull/388